lib/public_key/src/public_key.erl

```diff
@@ -2238,7 +2238,17 @@ cacerts_get() ->
 %%--------------------------------------------------------------------
 -doc(#{title => <<"Certificate API">>,
        since => <<"OTP 25.0">>}).
--doc "Loads the OS supplied trusted CA certificates.".
+-doc """
+Loads the OS supplied trusted CA certificates.
+
+This can be overridden by setting the `cacerts_path`
+environment key of the `public_key` application with
+the location of an alternative OS supplied certificate.
+You can set it via the command line as:
+
+    erl -public_key cacerts_path '"/path/to/certs.pem"'
+
+""".
 
```

Comment:
We can not really ensure that the path is to an alternative OS supplied certificate, but of course it is up to the user to ensure that their input is something that they can trust.  I am just pondering if it is clear enough. 

Maybe  we should say  something like  

"Loads the OS supplied trusted CA certificates form its standard location

The location may be customized using alternative OS specific paths set by the environment key `cacerts_path` of the public_key application. You can set it via the command line   ...  or in the system configuration file. (maybe add link to 
https://www.erlang.org/doc/system/applications.html#configuring-an-application)  Use with care, it up to the user to ensure that the certificates found in this alternative path can be trusted by the application it is configuring."